src/components/Attachments/AttachmentCarousel/index.tsx

```diff
@@ -59,7 +59,7 @@ function AttachmentCarousel({report, reportActions, parentReportActions, source,
         const parentReportAction = report.parentReportActionID && parentReportActions ? parentReportActions[report.parentReportActionID] : undefined;
         let targetAttachments: Attachment[] = [];
         if (type === CONST.ATTACHMENT_TYPE.NOTE && accountID) {
-            targetAttachments = extractAttachments(CONST.ATTACHMENT_TYPE.NOTE, {report, accountID});
+            targetAttachments = extractAttachments(CONST.ATTACHMENT_TYPE.NOTE, {privateNotes: report?.privateNotes, accountID});
```

Comment:
The useEffect dependencies have `report.reportID` and `report` replace them with `report.privateNotes`